pkg/terraform/schema/mixin.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I've made a document (not merged yet) that explains how to implement this command

https://github.com/deislabs/porter/pull/221

Feel free to review and leave comments if you like!

Reply:
Mind if we elaborate on this file in a follow up PR?

Reply:
Oh yeah sorry that wasn't clear. Everything is fine in follow-on PR. The only thing I really wanted in this one is cutting out k8s from vendor.